python/private/repo_utils.bzl

```diff
@@ -20,34 +20,34 @@ This code should only be loaded and used during the repository phase.
 REPO_DEBUG_ENV_VAR = "RULES_PYTHON_REPO_DEBUG"
```

Comment:
This file got `rctx` -> `mrctx` replacements for any function that supports both.

Reply:
Thanks!